tests/integration/test_ping.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pytest
+
+from tests.helpers import IntegrationTestCaseBase
+
+
+@pytest.mark.integration
+class TestPing(IntegrationTestCaseBase):
+    def setup_method(self):
+        super(TestPing, self).setup_method()
+        self.rebirthdb_host=os.getenv('REBIRTHDB_HOST')
```

Comment:
The `IntegrationTestCaseBase` already contains `os.getenv` to get the host. Instead of assign it to a variable in this test case, what do you think about extending the https://github.com/RebirthDB/rebirthdb-python/blob/master/tests/helpers.py#L14 to have a variable for this?

Reply:
Agreed.